poetry.lock

```diff
@@ -1651,13 +1651,13 @@ setuptools = "*"
 
 [[package]]
 name = "pep8-naming"
-version = "0.14.1"
+version = "0.15.0"
```

Comment:
pep8-naming 0.15.0 added type variable names checking. Now our linter step is failing with this:

```
Run poetry run ni-python-styleguide lint
./tests/component/test_stream_readers_di.py:182:2: N808 type variable name '_D' should use CapWords convention and an optional '_co' or '_contra' suffix
./tests/component/test_stream_readers_di.py:182:2: N808 type variable name '_D' should use CapWords convention and an optional '_co' or '_contra' suffix
Error: Process completed with exit code 255.
```

@bkeryan - what's the workflow here? Do we modify that code in a separate PR? Also with the pep8-naming update? Then what do we do with this PR?

Reply:
Oh, I see that box in the PR post now. Neat!